### DISCUSSION TITLE
cd to subdirectory of environment variable

### DISCUSSION THREAD

**Discussion by mvolkmann** [nushell/nushell#3173] [Q&A] [answered]
I have the environment variable `DOCUMENTS_DIR` set.
I can cd to that directory by entering `cd $nu.env.DOCUMENTS_DIR`.
But I can't cd to a subdirectory with this: `cd $nu.env.DOCUMENTS_DIR/blog`.
What is the correct syntax to do this?

**Comment by fdncred** [comment, marked as answer]
Something like this `ls $(echo $nu.env.HOME | path join Downloads)` would work.

**Comment by mercibac** [comment]
-- It's just an update --

With the actual version of **Nushell: 0.81.0** (in 2023) the method to cd into a subdirectory of an environment variable has changed a little. To achieve that now we can use something like: `cd $(echo $env.HOME | path join Downloads)`

**Reply by fdncred** [reply]
That syntax has changed. This would work now `cd ($env.HOME | path join Downloads)`

**Comment by Taywee** [comment]
If you want something more like what you're used to with POSIX shell, `cd $"($env.HOME)/Downloads"` will work too.
